drones/main.py

```diff
@@ -4,12 +4,13 @@
 """
 
 import logging
-from connection import c  # Sample library import
+from drones.connection import connector
 
 
 def setup_log():
     """Setup global log config."""
 
+    logging.getLogger()

```

Comment:
You create a logger, but don't assign it to anything. What's more, if you do it inside function, it will only be visible inside its scope.